doc/rados/operations/placement-groups.rst

```diff
@@ -88,7 +88,10 @@ number of PGs that the pool is working towards, if a ``pg_num``
 change is in progress).  **NEW PG_NUM**, if present, is what the
 system believes the pool's ``pg_num`` should be changed to.  It is
 always a power of 2, and will only be present if the "ideal" value
-varies from the current value by more than a factor of 3.
+varies from the current value by more than a factor of 3 by default.
+This factor can be be adjusted with::
+
+  ceph config set mgr mgr/pg_autoscaler/threshold 2.0
```

Comment:
Think this command is too long. Maybe, we can make use of the CLIWriteCommand and the function set_module_option(), e.g., https://github.com/ceph/ceph/blob/master/src/pybind/mgr/pg_autoscaler/module.py#L246

maybe change the command to: `ceph osd pool set threshold <num>`